docs/man/sat-jobstat.8.rst

```diff
@@ -0,0 +1,61 @@
+=============
+ SAT-JOBSTAT
+=============
+
+-------------------------
+Jobstat Short Description
+-------------------------
+
+:Author: Hewlett Packard Enterprise Development LP.
+:Copyright: Copyright 2022 Hewlett Packard Enterprise Development LP.
+:Manual section: 8
+
+SYNOPSIS
+========
+
+**sat** [global-opts] **jobstat** [options]
+
+DESCRIPTION
+===========
+
+Jobstat is a command that allows a user or admin to access application and job data via the command line. The command provides a table showing the user the apid, jobid, user, state, and time-reported of all jobs on the system.
+
+OPTIONS
+=======
+
+These options must be specified after the subcommand.
+
+**-h, --help**
+        Print the help message for 'sat jobstat'.
+
+**-a, --all**
+        Show all application information that is currently documented on the system.
+
+Summarize Remaining Options
+
+EXAMPLES
+========
+
+An example usage of the command:
```

Comment:
I would like to reword this so that it describes what the command does, i.e. "display a report of all jobs on the system"